appraisal.lic

```diff
@@ -29,6 +30,16 @@ class Appraisal
       pouch_adjective = settings.gem_pouch_adjective
     end
 
+    if args.count
+      if settings.low_value_gem_pouch_container.empty? || settings.low_value_gem_pouch_container.nil?
+        DRC.message("\"count\" passed as an argument, but no low_value_gem_pouch_container defined. Exiting") 
+        exit
```

Comment:
Is it better to exit if unable to count, or do as much as possible (i.e. appraise without counting)?

We could do this check on line 106-ish, output a message if we can't count, and then set `@count = false` (roughly). What do you think?

Reply:
Agreed. See current iteration. It's loud, but it works :) (It's loud in that it messages at every iteration that it can't count)